Vizzuality/gfw
# [FLG-627] Grants and fellowships projets pagination

## Overview

Wordpress by default only lets us get 100 posts at once. If G&F gets more Projects (it’s now at 94), they won’t load.

Instead of editing things so that the page fetches more project cards at a time, we’ve opted to paginate and load 21 posts at a time. We’ll follow the same styling that’s done on the GFW Blog (image below). Pagination should improve the loading speed of this page and probably help SEO-wise.

## Demo

![grants-and-fellowships](https://user-images.githubusercontent.com/23243868/220932925-21a66d38-8308-4a96-9e42-976fcb9d606f.png)

## Notes
To meet the criteria we had to change the way we provide filters/search in this page.

- From now on, both search and filter are requesting WP Rest API for new items instead just order/omit items on front. 
- Countries filter logic was moved to SSR to avoid these computation on frontend side. 
- WP Rest API was changed and its code is on Pantheon. We should deploy it after that deploy.

## Testing

- Connect to WordPress locally and make sure you have all changes to get the advanced search feature.
- Visit /grants-and-fellowships/projects/
